### PR TITLE
feat(intake-formats): migrate fortiweb to ingest

### PR DESCRIPTION
**Pull request by yosradirgham**
| Before update                                                           | After update                                               |
| ----------------------------------------------------------------------- | ---------------------------------------------------------- |
| `action.properties` is of type `list[dict]`                             | `action.properties` is of type `dict` (no impact expected) |
| `http.method: GET`                                                      | `http.method: GET` (no impact expected)                    |
| `source.port`, `http.request.bytes`, `http.response.bytes` type `float` | type int                                                   |

Parent Task: [#41572](https://github.com/SekoiaLab/platform/issues/41572)

### CODE REVIEW

File: Fortinet/fortiweb/ingest/parser.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: FortiWeb
+pipeline:
+  - name: parsed_event
+    external:
+      name: kv.parse-kv
+      properties:
+        value_sep: "="
```
Comment (PierrickV) [inline Fortinet/fortiweb/ingest/parser.yml:7]:
```suggestion        
        value_sep: "="
        item_sep: \s
```

File: Fortinet/fortiweb/_meta/fields.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+tmp.device_id:
+  name: tmp.device_id
+  description: "The device identifier"
+  type: keyword
+
+tmp.log_id:
+  name: tmp.log_id
+  description: "The log id"
+  type: keyword
```
Comment (squioc) [inline Fortinet/fortiweb/_meta/fields.yml:9]: These fields are useless
```suggestion
```

File: Fortinet/fortiweb/ingest/parser.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          http.version: "{{parsed_event.message.http_version}}"
+          log.level: "{{parsed_event.message.pri}}"
+          network.protocol: "{{parsed_event.message.proto}}"
+          rule.ruleset: "{{parsed_event.message.policy}}"
+          source.geo.name: "{{parsed_event.message.srccountry}}"
+          source.ip: "{{parsed_event.message.src}}"
+          source.port: "{{parsed_event.message.src_port}}"
+          tls.cipher: "{{parsed_event.message.cipher_suite}}"
+          tmp.device_id: "{{parsed_event.message.device_id}}"
+          tmp.log_id: "{{parsed_event.message.time}}"
```
Comment (squioc) [inline Fortinet/fortiweb/ingest/parser.yml:35]:
```suggestion
          device.id: "{{parsed_event.message.device_id}}"
```

**Review by squioc — CHANGES_REQUESTED** [review]

**Review by squioc — APPROVED** [review]
LGTM

**Review by squioc — APPROVED** [review]

**Pull request merged**